vizro-ai/docs/pages/user-guides/create-complex-dashboard.md

```diff
@@ -0,0 +1,149 @@
+# Generate a complex dashboard
+
+This guide shows you how to instruct Vizro-AI generating a complex dashboard.
+
+Vizro-AI can in general follow user requirements well and generate high-quality dashboards, but the nature of LLMs means that generated dashboards are not always an exact match for user expectations. When the text length of user requirements increases, the LLMs could start to miss part of the user requirements or make mistakes. Apart from choosing more advanced models for harder tasks, improving the user prompt could help too.
+
+The following example shows how to use Vizro-AI to generate a complex Vizro dashboard.
+
+## 1. Install Vizro-AI and its dependencies
+
+If you haven't already installed Vizro-AI and set up the API key for OpenAI, follow the [installation guide](../user-guides/install.md).
+
+
+## 2. Open a Notebook
+A good way to initially explore Vizro-AI is from inside a Jupyter Notebook.
+
+??? "If you haven't used Jupyter before..."
+
+    You may need to install the Jupyter package if you . From the terminal window:
+
+    ```bash
+    pip install jupyter
+    ```
+
+Start a new Notebook as follows:
+
+```bash
+jupyter notebook
+```
+
+The command opens Jupyter in a browser tab. Use the UI to navigate to a preferred folder in which to create this new dashboard.
+
+Create a new `Python 3 (ipykernel)` Notebook from the "New" dropdown. Confirm your Vizro installation by typing the following into a cell in the Notebook and running it.
+
+```py
```

Comment:
Would scrap those two sections

Reply:
Why? This is a tool for inexperienced folks and they may not have installed Vizro, so won't have had this information. I think it's key to onboarding some people and I don't want to send them elsewhere to get the instructions (not even over to Vizro) since that's confusing.

Reply:
I thought the idea of the "How to" guides is to explain one specific task alone. We already have a "How to" guide on installation, and we also have a tutorial which mimics that.

> In how-to guides, practical usability is more helpful than completeness. Whereas a tutorial needs to be a complete, end-to-end guide, a how-to guide does not. It should start and end in some reasonable, meaningful place, and require the reader to join it up to their own work.

> A how-to guide serves the work of the already-competent user, whom you can assume to know what they want to do, and to be able to follow your instructions correctly.

From https://diataxis.fr/how-to-guides/

Reply:
Ah sorry, I thought this was in the tutorials section for some reason, my bad. Agree that it's not the way for how-tos, and also that if you're looking at advanced (complex) dashboards, it's not your first project, so yes, will remove it. I wonder how it got there...maybe I added to the wrong file 🤔